app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
+
+  def recent_posts
+    posts.first(3)
```

Comment:
- [ ] In as much as this approach works for in memory data, is not advisable to be used in database queries. Kindly make changes to avoid seeing unexpected results. __HINT:__  `posts.order('created_at Desc').limit(3)`